tests/test_astrometryTask.py

```diff
@@ -211,6 +222,18 @@ def makeSourceCat(self, distortedWcs):
             src.set(sourceCentroidKey, refObj.get(refCentroidKey))
             src.set(sourceInstFluxKey, refObj.get(refFluxRKey))
             src.set(sourceInstFluxErrKey, refObj.get(refFluxRKey)/100)
+
+        # Make a few outlier fluxes for the outlier rejection tests.
+        # Otherwise, these fluxes are not used.
+        unResolved, = np.where(~refCat['resolved'])
+
+        sourceCat[sourceInstFluxKey][unResolved[0]] = 0.0
+        sourceCat[sourceInstFluxKey][unResolved[1]] = -1.0
+        sourceCat[sourceInstFluxKey][unResolved[2]] *= 1000.0
```

Comment:
A comment would be good here, to note what each of these values should be testing. I think `0` and `-1` are to check the nan/inf rejection math, while 1000 is to check the outlier reject level?